src/ontology/uberon-edit.obo

```diff
@@ -157956,7 +157956,7 @@ xref: DHBA:10431
 xref: FMA:76989
 xref: PBA:128013078
 intersection_of: UBERON:0013605 ! layer of lateral geniculate body
-intersection_of: composed_primarily_of CL:0011003 ! magnocellular neurosecretory cell (duplicate class)
+intersection_of: composed_primarily_of CL:0011003 ! magnocellular neurosecretory cell
```

Comment:
not 100% sure why this change suddenly happened, but doesn't seem like a bad one